src/main/resources/view/ModuleListCard.fxml

```diff
@@ -9,33 +9,29 @@
 <?import javafx.scene.layout.Region?>
 <?import javafx.scene.layout.RowConstraints?>
 <?import javafx.scene.layout.VBox?>
-<HBox xmlns:fx="http://javafx.com/fxml/1" id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/11.0.1">
+<HBox
+        stylesheets="@DarkTheme.css"
+        xmlns:fx="http://javafx.com/fxml/1" id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/11.0.1">
     <GridPane HBox.hgrow="ALWAYS">
         <columnConstraints>
-            <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150"/>
+            <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="100"/>
         </columnConstraints>
-        <VBox alignment="CENTER_LEFT" minHeight="105" spacing="3.0" GridPane.columnIndex="0">
+        <VBox styleClass="module-card" alignment="CENTER_LEFT" spacing="1.5" GridPane.columnIndex="0">
             <padding>
-                <Insets bottom="5" left="15" right="5" top="5"/>
+                <Insets bottom="2" left="8" right="2" top="2"/>
             </padding>
-            <Label fx:id="moduleCode" style="-fx-font-size: 28; -fx-text-fill: #FF7A2F;" styleClass="module-cell-title"
-                   text="\$moduleCode"/>
-            <Label fx:id="title" styleClass="module-cell-title" text="\$first"/>
+            <Label fx:id="moduleCode" styleClass="module-code" text="\$moduleCode"/>
+            <Label wrapText="true" fx:id="title" styleClass="title" text="\$first"/>
             <HBox spacing="10.0">
                 <children>
-                    <Label fx:id="semester" styleClass="cell_small_label" text="Y?S?"/>
-                    <Region minHeight="-Infinity" prefWidth="1.0" style="-fx-border-color: #3d3d3d;"/>
-                    <Label fx:id="credits" styleClass="module-cell-default" text="\$credits"/>
-                    <Region prefWidth="1.0" style="-fx-border-color: #3d3d3d;"/>
-                    <Label styleClass="module-cell-default" text="Grade: A"/>
+                    <Label styleClass="module-card-default" fx:id="semester" text="Y?S?"/>
+                    <Region styleClass="vertical-divider"/>
+                    <Label styleClass="module-card-default" fx:id="credits" text="\$credits"/>
+                    <Region styleClass="vertical-divider"/>
+                    <Label styleClass="module-card-default" text="Grade: A"/>
```

Comment:
@waynewee you can change this to the actual module grade instead of the placeholder, grade is working now